### PR TITLE
Add shareable text SVG for YIM stats 

### PR DESCRIPTION
**Pull request by amCap1712**
For testing, see

- https://beta-api.listenbrainz.org/1/art/year-in-music/2022/aerozol?image=stats
- https://beta-api.listenbrainz.org/1/art/year-in-music/2022/aerozol?image=tracks
- https://beta-api.listenbrainz.org/1/art/year-in-music/2022/aerozol?image=albums

Reference:
https://usercontent.irccloud-cdn.com/file/puoSTsv4/image.png

### CODE REVIEW

**Comment by amCap1712** [comment]
Update most listened day messages:

I survived Mondays with music
I chilled with music on Tuesdays
I got through Wednesdays with music
I spent time with my tunes on Thursdays
I celebrated Fridays with music
I partied hard - or hardly! - with music on Saturdays
I loved spending Sundays with music

**Review by mayhem — APPROVED** [review]
It is impressive that github rendered that SVG file as well as it did!

File: listenbrainz/webserver/views/art_api.py [hunk 1]
```diff
@@ -216,3 +219,91 @@ def cover_art_custom_stats(custom_name, user_name, time_range, image_size):
...
+        bg_image_url=f'{current_app.config["SERVER_ROOT_URL"]}/static/img/art/yim-2022-shareable-bg.png',
+        stereo_image_url=f'{current_app.config["SERVER_ROOT_URL"]}/static/img/art/yim-2022-shareable-stereo.png',
+    )
+
+
+
+@art_api_bp.route("/year-in-music/2022/<user_name>", methods=["GET"])
+@crossdomain
+@ratelimit()
+def cover_art_yim_2022(user_name):
```
Comment (mayhem) [inline listenbrainz/webserver/views/art_api.py:283]: My only concern here is that this endpoint has the year in it, but none of the supporting functions do. I suppose if we change nothing for 2023, then we're fine, but I doubt we'll be doing that. Not urgent right now, but something to think about...

**Review by mayhem — APPROVED** [review]

File: listenbrainz/webserver/views/art_api.py (hunk 1 above)
Comment (amCap1712) [reply listenbrainz/webserver/views/art_api.py:283]: I think at the least the theme of the svgs would change next year and we would have to create new templates and supporting functions then anyway.

**Review by MonkeyDo — APPROVED** [review]
As explained on IRC, playlist images are pending actually generating and importing the playlists.

The rest is  * *chef's kiss* *

**Pull request merged**